pkg/messaging/v1/invoke_method_response.go

```diff
@@ -58,8 +58,9 @@ func (imr *InvokeMethodResponse) WithMessage(pb *commonv1pb.InvokeResponse) *Inv
 func (imr *InvokeMethodResponse) WithRawData(data []byte, contentType string) *InvokeMethodResponse {
 	imr.r.Message.ContentType = contentType
 
-	// Clone data to prevent GC from deallocating data
-	imr.r.Message.Data = &anypb.Any{Value: cloneBytes(data)}
```

Comment:
This was an interesting find. One of the main ways fasthttp is optimized is that it reduces allocations to almost zero, so it reuses buffers. Interestingly, Dapr was then cloning the data into a new byte slice (otherwise the buffer would be reused by fasthttp), sort-of negating the positive effects of fasthttp.